auxx/src/Mode.hs

```diff
@@ -183,29 +180,14 @@ instance {-# OVERLAPPING #-} CanJsonLog AuxxMode where
 instance HasConfiguration => MonadDBRead AuxxMode where
     dbGet = realModeToAuxx ... dbGet
     dbIterSource tag p = hoist (hoist realModeToAuxx) (dbIterSource tag p)
+    dbGetRawBlock = realModeToAuxx ... DB.dbGetRawBlockRealDefault
```

Comment:
This should be `dbGetRawBlock = realModeToAuxx ... dbGetRawBlock` instead. Note that we don't want to choose the `Default` implementation here — if another implementation is used in `Real` mode, we won't have to remember to update `auxx` accordingly.

Reply:
it's already fixed

Reply:
can you take away your diss?

Reply:
It's not fixed as far as I can see.